### PR TITLE
CLOUD-374 Fix proxysql pool query

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/proxysql_exporter?pullRequest=28) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/proxysql_exporter?pullRequest=28) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/proxysql_exporter?pullRequest=28) it.</sub>

**Comment by rnovikovP** [comment]
@cap1984  pls click on the  CLA button to make CLA checks green. Thanks

File: exporter.go [hunk 1]
```diff
@@ -267,7 +267,7 @@ func scrapeMySQLGlobal(db *sql.DB, ch chan<- prometheus.Metric) error {
 	return rows.Err()
 }
 
-const mySQLconnectionPoolQuery = "SELECT hostgroup, srv_host, srv_port, * FROM stats_mysql_connection_pool"
```
Comment (AlekSi) [inline exporter.go]: Code below assumes that those three columns are in that order: https://github.com/percona/proxysql_exporter/blob/90da3e31c43d9eed7575b4950162840723baadc9/exporter.go#L314-L320

There is a Docker image that can be used for testing: https://hub.docker.com/r/percona/proxysql2

Will you work on it? If not, who will?

Comment (AlekSi) [reply exporter.go]: Here is a source repo for Dockerfile: https://github.com/percona/percona-docker/tree/master/proxysql

**Comment by AlekSi** [comment]
And to close a loop:

> time="2019-09-06T11:06:11Z" level=info msg="time=\"2019-09-06T11:06:11Z\" level=error msg=\"Error scraping for collect.mysql_connection_pool: Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '* FROM stats_mysql_connection_pool' at line 1\" source=\"exporter.go:176\"" agentID=/agent_id/0bf1148f-0dcf-46f0-a610-18636cbd7db2 component=agent-process type=proxysql_exporter

That request should never reach MySQL. I guess you use "user" ProxySQL port, not admin port. That query works on ProxySQL.

**Comment by cap1984** [comment]
Please disregard such an issue.
It happened due to incorrect port specified as ProxySQL admin port

**Pull request closed without merging**